protostar/commands/test/starkware/cheatable_state.py

```diff
@@ -119,6 +120,9 @@ def __init__(self, *args, **kwargs):
             AddressType, Dict[SelectorType, List[int]]
         ] = defaultdict(dict)
         self.event_selector_to_name_map: Dict[int, str] = {}
+        self.class_hash_to_contract_path_map: Dict[int, Path] = {}
+        self.contract_address_to_contract_path_map: Dict[int, Path] = {}
```

Comment:
Can we merge those two dicts into one as they have the same key. We won't have to rememeber about inserting into two dicts.

Reply:
Not really. This map is mainly used by syscall `deploy`.